lib/erl_type.ml

```diff
@@ -151,47 +155,55 @@ let rec of_etf = function
      | AtomTag ->
         begin match elements with
         | Etf.Atom "any" ->
-           Ok (Atom ["any"])
+            Ok (Atom {atoms_union=["any"]})
```

Comment:
Sorry this is a bug from before: this case is `atom()` not `'any'`.
Could you add a type represent `AtomsUnion` or `AnyAtom` and fix it?

Reply:
I fixed to handle `atom()` type correctly.